telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/command/autocomplete/AutocompleteHandler.java

```diff
@@ -42,8 +42,14 @@ public boolean onMessage(Long chatId, String message, Update update) {
         try {
             if (autocompletes.containsKey(message)) {
                 for (String text: autocompletes.get(message)){
-                    botApi.sendMessage(chatId, text);
-                    Thread.sleep(400);
+                    if(text.startsWith("stickerId:")){
+                        botApi.sendSticker(chatId.toString(), text.replace("stickerId:",""));
```

Comment:
(optional) sendSticker should take Long argument for chatId. Can you please correct this as well?